esp-hal/src/soc/esp32/gpio.rs

```diff
@@ -231,29 +226,6 @@ pub enum InputSignal {
     PWM1_CAP0             = 112,
     PWM1_CAP1             = 113,
     PWM1_CAP2             = 114,
-    PWM2_FLTA             = 115,
```

Comment:
Did you mean to remove these?

Reply:
yup, they aren't present in the TRM and they aren't used in the HAL.